modules/manage/pages/manage-security/configure-saml.adoc

```diff
@@ -0,0 +1,198 @@
+* *Current SP consume URL*: the URL the IdP sends the user to after they have authenticated.
+* *Current SP logout URL*: the URL to which the IdP sends the user when they log out of the Web Console.
+
+=== Configure Couchbase Metadata
+
+Under *Metadata*, enter information about your Couchbase Server and organization. Most of these fields supply contact information to your IdP and do not affect the authentication process. The exceptions are:
+
+SP Entity ID::
+A URL that identifies your Couchbase Server to the IdP. You add this URL to your IdP's configuration so it can identify which service the user is authenticating with. This field is optional. If you do not enter a value, Couchbase Server uses a URL based on the node's fully qualified domain name, port number, and the path to the metadata page (`/saml/metadata`). 
+[[sp-base-url]]
+SP Base URL Type::
+This option controls hostname in the URL that the IdP redirects the user's browser to after authentication. Choose this option based on the address users enter to connect to your Couchbase Server:
++
+* *Node address* sends the user back to the self-reported address of the node the user is logging into. 
+* *Alternate node address* sends the user to the node's alternate address. See xref:learn:clusters-and-availability/connectivity.adoc#alternate-addresses[Alternate Addresses].
+* *Custom URL* sends the user to the URL you enter in *Custom URL* which appears when you select this option. Select this option when users use some other URL when connecting to nodes in the cluster. For example, if you use a load balancer to redirect users to nodes, enter its URL here. 
+
+Key and Certificates::
+Add the Couchbase Server's private key and certificate. Either copy and paste their content or upload them by clicking bth:[Select File]. If the certificate was not directly signed by a CA, added the intermediate certificates in *Certificate chain*. 
+
+### Configure the IdP
+
+Under *Identity Provider Configuration*, enter the information about your IdP: 
+
+Load IDP metadata from::
+Choose to either provide the IdP's metadata URL or to upload a snapshot of its metadata in a file. If you provide a URL, you can have Couchbase Server refresh the metadata periodically.
+Verify remote peer::
+Select to enable verification of the IdP's certificate. If you enable this option, supply a copy of the IdP's certificate in *CA Certificates*.   
+Validate metadata using trusted fingerprints::
+Select to enable using fingerprints instead of a certificate to authenticate the metadata from the IdP. Select *Use trusted fingerprints for metadata bootstrap only* to limit using fingerprint verification to just the initial retrieval of the metadata. Select *Always use trusted fingerprints to validate signatures in SAML...* to use fingerprints to validate all SAML messages instead of just metadata.
```

Comment:
> Select to enable using fingerprints instead of a certificate to authenticate the metadata from the IdP

"to authenticate the metadata from the IdP" -> "validate signature of the metadata from the IdP"

Please also drop "instead of a certificate", because we can use https with remote peer verification and validate metadata signature at the same time.